src/data/GraphOnChip.ts

```diff
@@ -757,22 +758,93 @@ export default class GraphOnChip {
         return [...nodes];
     }
 
-    getAllLinks(): NetworkLink[] {
-        const links: NetworkLink[] = [];
+    getAllLinksInitialState() {
+        const links: Record<
```

Comment:
`links` is non descriptive, there shoudl be a mention of what those are recoreded by